test/KillSwitchOracle.t.sol

```diff
@@ -0,0 +1,393 @@
+            liquidationThreshold: 0,
+            liquidationBonus:     0
+        });
+
+        assertEq(pool.getReservesList().length, 5);
+
+        vm.prank(owner);
+        killSwitchOracle.setOracle(address(oracle), 0.99e8);
+        oracle.setLatestAnswer(0.98e8);
+
+        vm.expectEmit(address(killSwitchOracle));
+        emit Trigger(address(oracle), 0.99e8, 0.98e8);
+        vm.expectEmit(address(killSwitchOracle));
+        emit AssetLTV0(asset1);
+        vm.expectEmit(address(killSwitchOracle));
+        emit AssetFrozen(asset2);
+        vm.prank(randomAddress);  // Permissionless call
+        killSwitchOracle.trigger(address(oracle));
+
+        _assertReserve({
+            asset:                asset1,
+            active:               true,
+            frozen:               false,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 83_00,
+            liquidationBonus:     105_00
+        });
+
+        _assertReserve({
+            asset:                asset2,
+            active:               true,
+            frozen:               true,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 0,
+            liquidationBonus:     0
+        });
+
+        _assertReserve({
+            asset:                asset3,
+            active:               true,
+            frozen:               true,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 25_00,
+            liquidationBonus:     110_00
+        });
+
+        _assertReserve({
+            asset:                asset4,
+            active:               true,
+            frozen:               false,
+            paused:               true,
+            ltv:                  80_00,
+            liquidationThreshold: 83_00,
+            liquidationBonus:     105_00
+        });
+
+        _assertReserve({
+            asset:                asset5,
+            active:               false,
+            frozen:               false,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 0,
+            liquidationBonus:     0
+        });
+    }
+
+    function _initReserve(
+        address asset,
+        bool active,
+        bool frozen,
+        bool paused,
+        uint256 ltv,
+        uint256 liquidationThreshold,
+        uint256 liquidationBonus
+    ) internal {
+        DataTypes.ReserveConfigurationMap memory configuration = pool.getConfiguration(asset);
+        configuration.setActive(active);
+        configuration.setFrozen(frozen);
+        configuration.setPaused(paused);
+        configuration.setLtv(ltv);
+        configuration.setLiquidationThreshold(liquidationThreshold);
+        configuration.setLiquidationBonus(liquidationBonus);
+        pool.__addReserve(asset, configuration);
+    }
```

Comment:
I'd add these two extra lines for readability
```suggestion
        DataTypes.ReserveConfigurationMap memory configuration = pool.getConfiguration(asset);

        configuration.setActive(active);
        configuration.setFrozen(frozen);
        configuration.setPaused(paused);
        configuration.setLtv(ltv);
        configuration.setLiquidationThreshold(liquidationThreshold);
        configuration.setLiquidationBonus(liquidationBonus);

        pool.__addReserve(asset, configuration);
    }
```

Reply:
done